README.md

```diff
@@ -1,12 +1,38 @@
 # Resistance Calendar
 
-## Setup
+> PRs welcome; seeking all roles
+
+See our [contributing guide](CONTRIBUTING.md) for more information.
+
+*The Resistance Calendar* aims to empower progressive activists by providing a massive central listing of upcoming events. Crowdsourced event submissions are curated by a team of volunteers, while events from allied organizations will be included automatically. Event filters make it easy for people to find events near them. Filtered notifications help people discover new events immediately. 
+
+This is a rebuild of the live website currently found [here](https://www.resistancecalendar.org/).
+
+## About the stack
+
+This repository covers the ReactJS front-end.  Additional parts of the stack can be found on our [organizational page](https://github.com/ResistanceCalendar).
+
+- [API](https://github.com/ResistanceCalendar/resistance-calendar-api)
+- [Chrome Plugin](https://github.com/ResistanceCalendar/resistance-calendar-chrome-ext)
+- [Facebook Feed](https://github.com/ResistanceCalendar/FacebookFeed)
+- [Calendar Watcher](https://github.com/ResistanceCalendar/resistance-cal-watcher)
+- [Drupal](https://github.com/ResistanceCalendar/resistance-drupal)
```

Comment:
I thinkhttps://github.com/ResistanceCalendar/resistance-calendar-api will be the only other relevant repo.